python/setup.py

```diff
@@ -0,0 +1,50 @@
+import os
+from setuptools import setup
+
+
+def package_files(directory):
+    paths = []
+    for (path, directories, filenames) in os.walk(directory):
+        for filename in filenames:
+            paths.append(os.path.join('..', path, filename))
+    return paths
+
+
+java_stubs_paths = package_files('java-stubs') + ['../py.typed']
+jpype_stubs_paths = package_files('jpype-stubs') + ['../py.typed']
+org_stubs_paths = package_files('org-stubs') + ['../py.typed']
+
+setup(name='cogroo4py',
+      version='0.4.3',
+      description='Interface for accessing CoGrOO from Python scripts using jpype',
+      author='Guilherme Passero',
```

Comment:
You have made a huge contribution to this repo so if you agree please add yourself here as an author, or at least a contributor 🙂